alcs-frontend/src/app/features/notice-of-intent/proposal/parcel-prep/parcel-prep.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { Router } from '@angular/router';
+import { NoticeOfIntentParcelService } from '../../../../services/notice-of-intent/notice-of-intent-parcel/notice-of-intent-parcel.service';
+
+@Component({
+  selector: 'app-parcel-prep',
+  templateUrl: './parcel-prep.component.html',
+  styleUrls: ['./parcel-prep.component.scss']
+})
+export class ParcelPrepComponent implements OnChanges {
+  @Input() fileNumber = '';
+
+  displayedColumns = ['number', 'pid', 'pin', 'civicAddress', 'area', 'alrArea', 'owners', 'actions'];
+  parcels: {
+    pin?: string;
+    pid?: string;
+    mapAreaHectares: string;
+    alrArea: number;
+    owners: string;
+    fullOwners: string;
+    hasManyOwners: boolean;
+    uuid: string;
+  }[] = [];
+
+  constructor(private parcelService: NoticeOfIntentParcelService, private router: Router) {}
+
+  async loadParcels(fileNumber: string) {
+    const parcels = await this.parcelService.fetchParcels(fileNumber);
+    this.parcels = parcels.map((parcel) => ({
+      ...parcel,
+      owners: `${parcel.owners[0].displayName} ${parcel.owners.length > 1 ? ' et al.' : ''}`,
```

Comment:
The logic for this is a bit more complex on how the backend calculates it. Its the first owner alphabetically with numbers coming last. I wonder if we will get a bug where the owner name here doesn't match the other pages?

Reply:
Discussed: this one will be addressed in separate ticket since it is also true for applications